packages/app/src/runner/selector-playground/SelectorPlayground.vue

```diff
@@ -1,24 +1,8 @@
 <template>
   <div
     id="selector-playground"
-    class="border-t border-b bg-gray-1000 border-gray-800 h-[56px] grid py-[12px] px-[16px] gap-[12px] grid-cols-[40px,1fr,auto] items-center "
+    class="border-t border-b bg-gray-1000 border-gray-800 h-[56px] grid py-[12px] px-[16px] gap-[12px] grid-cols-[1fr,auto] items-center "
   >
-    <SelectorPlaygroundTooltip
-      color="light"
-      placement="top"
-      :hover-text="t('runner.selectorPlayground.playgroundTooltip')"
-      class="flex h-full"
-    >
-      <button
-        class="border rounded-md flex h-full outline-solid outline-indigo-500 transition w-[40px] duration-150 items-center justify-center hover:bg-gray-800"
-        :aria-label="selectorPlaygroundStore.isEnabled ? 'click to interact with the application and build test cases' : 'click to exit interactive test building mode'"
-        data-cy="playground-toggle"
-        :class="{ 'bg-gray-800 border-gray-700': selectorPlaygroundStore.isEnabled, 'bg-gray-900 border-gray-800': !selectorPlaygroundStore.isEnabled }"
-        @click="toggleEnabled"
-      >
-        <i-cy-selector_x16 class="icon-dark-gray-300" />
-      </button>
-    </SelectorPlaygroundTooltip>
```

Comment:
We talked about removing this button as its redundant. The work wasn't done since the assumption was we were getting rid of it.